client/package.json

```diff
@@ -29,7 +29,8 @@
     "semantic-ui-css": "^2.4.1",
     "semantic-ui-react": "^0.87.3",
     "showdown": "^1.9.0",
-    "validator": "^11.1.0"
+    "validator": "^11.1.0",
```

Comment:
I think you was deprecated this package by using `yup`

Reply:
fixed